src/data/set/pointwise.lean

```diff
@@ -863,6 +863,10 @@ image2_Inter₂_subset_right _ _ _
 
 @[to_additive] lemma finite.smul : s.finite → t.finite → (s • t).finite := finite.image2 _
 
+@[simp, to_additive] lemma bUnion_smul_set (s : set α) (t : set β) :
+  (⋃ a ∈ s, a • t) = s • t :=
+Union_image_left _
```

Comment:
What about the right variant of this?

Reply:
It does not make sense (and your statement does not typecheck) because we can multiply `a : α` by `s : set α` only on the left. See https://github.com/leanprover-community/mathlib/pull/14696/files/4b7d438631547104a060f213d23c5c124206001d#diff-3efc1e7d1974f57473e535503759592ac5bfcf4ae8c7d83a7ecc6fbb6d4ef261